### PR TITLE
Update generated docs + Add tutorials page link to docs

### PR DESCRIPTION
**Pull request by FlorentinD**
- Improve title for FastRP + KNN example
- Update generated docs
- Link to generated notebook docs

Thank you for your contribution to the Graph Data Science Client project.

Before submitting this PR, please read [Contributing to the Neo4j Ecosystem](https://github.com/neo4j/graph-data-science-client/blob/main/CONTRIBUTING.md). 

Make sure:
- [x] You signed the [Neo4j CLA](https://neo4j.com/developer/cla/#sign-cla) (Contributor License Agreement) so that we are allowed to ship your code in our library
- [x] Your contribution is covered by tests

### CODE REVIEW

File: examples/fastrp-and-knn.ipynb [hunk 1]
```diff
@@ -386,8 +386,19 @@
...
+   "name": "python3"
+  },
   "language_info": {
-   "name": "python"
+   "name": "python",
+   "version": "3.10.6"
+  },
+  "vscode": {
+   "interpreter": {
+    "hash": "916dbcbb3f70747c44a77c7bcd40155683ae19c65e1c03b4aa3499c5328201f1"
```
Comment (FlorentinD) [inline examples/fastrp-and-knn.ipynb:400]: these got added automatically :/

**Review by adamnsch — APPROVED** [review]
Great :) I'll not forget about this next time!

Btw, we added links to the notebooks in our GDS server manual recently, eg: https://neo4j.com/docs/graph-data-science/current/machine-learning/node-embeddings/fastrp/

Should we update those titles as well? And should they point to the client docs now instead of the notebooks?

**Comment by FlorentinD** [comment]
We should have tooling to check these are in sync. When talking with Nicola, I think he wanted to give it a try as well.
The tooling is likely quite similar to what you added in #272

I would lean towards linking to the client docs there as well :thinking:

**Comment by adamnsch** [comment]
> We should have tooling to check these are in sync. When talking with Nicola, I think he wanted to give it a try as well. The tooling is likely quite similar to what you added in #272
> 
> I would lean towards linking to the client docs there as well thinking But we first need to publish the client docs, maybe after the 1.6 cherry pick is live.

Ah, that's a good idea. Should be easy to set up

**Pull request merged**